core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -255,12 +255,12 @@ object KafkaConfig {
   val LogFlushIntervalMsProp = "log.flush.interval.ms"
   val LogFlushOffsetCheckpointIntervalMsProp = "log.flush.offset.checkpoint.interval.ms"
   val LogPreAllocateProp = "log.preallocate"
-  val MessageFormatVersionProp = "message.format.version"
+  val LogMessageFormatVersionProp = "log.message.format.version"
```

Comment:
@gwenshap After taking another look at the current code. It seems the naming is not as simple as a prefix + config name. Many configuration names are not changed in the LogConfig. For example,
```log.roll.ms```, ```log.roll.jitter.ms``` and ```log.index.size.max.bytes``` in the KafkaConfig become ```segment.ms```, ```segment.jitter.ms``` and ```segment.index.bytes``` in the LogConfig.

Also, quite a few ```log.*``` configurations in KafkaConfig are not in LogConfig.

As of now I think we'd better just keep it as is but only make the new configurations using the log prefix + configuration name.